src/sxastarter/src/components/Navigation.tsx

```diff
@@ -96,14 +96,10 @@ export const Default = (props: NavigationProps): JSX.Element => {
           checked={isOpenMenu}
           onChange={() => handleToggleMenu()}
         />
-        <div className="menu-humburger">
-          <span></span>
-          <span></span>
-          <span></span>
-        </div>
```

Comment:
These changes are customizations. My bad for not adding a comment.

Reply:
Fixed